sigopt_sklearn/search.py

```diff
@@ -227,9 +243,9 @@ def _convert_log_params(self, param_dict):
       log_converted_dict = {}
       for pname in param_dict:
         pval = param_dict[pname]
-        if "__log__" in pname:
+        if '__log__' in pname:
           pval = math.exp(pval)
-          pname = pname.replace("__log__", "")
+          pname = pname.replace('__log__', '')
         log_converted_dict[pname] = pval
       return log_converted_dict
```

Comment:
+@iandewanker, do you know?

On Tue, Sep 13, 2016 at 4:02 PM, samuela <notifications@github.com> wrote:

> In sigopt_sklearn/search.py
> <https://github.com/sigopt/sigopt_sklearn/pull/3#discussion_r78660750>:
>
> >            pval = math.exp(pval)
> > -          pname = pname.replace("__log__", "")
> > +          pname = pname.replace('__log__', '')
> >          log_converted_dict[pname] = pval
> >        return log_converted_dict
>
> I have no idea. I just changed the string literals.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigopt/sigopt_sklearn/pull/3/files/c4aebe399cde9c812b67ed1062b9e13e65de5b45#r78660750>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH4nmJjIRXYtLU14-FVgKox0ywMpZQUmks5qpyuBgaJpZM4J8MEo>
> .
>